src/module/hook/user/index.ts

```diff
@@ -144,14 +145,32 @@ export function useUpdateUser() {
   const queryClient = useQueryClient();
   const navigate = useNavigate();
 
-  return useMutation<IUser[], any, IUpdateUser>(
+  return useMutation<UpdateStatus[], any, IUpdateUser>(
     (userUpdate: IUpdateUser) => {
       return api.API.User.Update(userUpdate);
     },
     {
       onSuccess: (data, userUpdate) => {
-        // Invalidate and refetch
-        queryClient.invalidateQueries("users");
+        if (
```

Comment:
Optimization for updating update viewed status without invalidating everything (showing flash of unloaded state).